starport/pkg/xgenny/run.go

```diff
@@ -0,0 +1,34 @@
+package xgenny
+
+import (
+	"context"
+
+	"github.com/gobuffalo/genny"
+	"github.com/gobuffalo/logger"
+	"github.com/tendermint/starport/starport/pkg/placeholder"
+)
+
+func DryRunner(ctx context.Context) *genny.Runner {
+	runner := genny.DryRunner(context.Background())
+	runner.Logger = logger.New(genny.DefaultLogLvl)
+	return runner
+}
+
+func RunWithValidation(tracer *placeholder.Tracer, gens ...*genny.Generator) error {
+	run := func(runner *genny.Runner, gen *genny.Generator) error {
+		runner.With(gen)
+		return runner.Run()
+	}
+	for _, gen := range gens {
+		if err := run(DryRunner(context.Background()), gen); err != nil {
```

Comment:
I thought we were replacing DryRunner with using a temporary dir

Have you solved the ibc module generation with another way?

Reply:
i think it is fine to merge this change without atomicity. important that we won't generate invalid code.
I will add a temporary directory probably tomorrow